### ISSUE TITLE
Where is the extra level for Self Practice Exercise 1-4?

### ISSUE THREAD

**Issue by zhuming15** [nus-cs2030/2223-s1#326]
Hi all, did you all manage to see the extra level of self-practice exercises 1-4? I thought Prof reopened and updated the exercises.

**Comment by tranquilityyy** [comment]
Once you re-submit the files that you have previously submitted (and cleared the exercises with), your grade will turn from "A" to C on codecrunch. There's no specific questions the extra level, but codecrunch will tell you what is still lacking (e.g. replacing all for loops / ifs in your code).

**Comment by ZYN2122** [comment]
If we get an A for the Exercises here, can we replace the recorded grade previously? I didnt manage to get all the As before the exercises were closed

**Comment by Wenbo-J** [comment]
Nope, there are no extra levels, it's reopened just for us to reattempt the exercises using Streams

**Comment by Penghao-99** [comment]
There are no extra levels or extra test cases. In the re-opened exercises, we are supposed to replace all if-else loops and for loops with streams.
